doc/transform_methods.rst

```diff
@@ -5,19 +5,23 @@ Transform Methods
 
-    2. The :doc:`Hansen–Law <transform_methods/hansenlaw>` recursive method of Hansen and Law, which provides an extremely fast transform with low centerline noise. 
+    2. The :doc:`hansenlaw <transform_methods/hansenlaw>` recursive method of Hansen and Law, which provides an extremely fast transform with low centerline noise. 
 
-    3. The :doc:`Direct <transform_methods/direct>` numerical integration of the analytical Abel transform equations, which is implemented in Cython for efficiency. In general, while the forward Abel transform is useful, the inverse Abel transform requires very fine sampling of features (lots of pixels in the image) for good convergence to the analytical result, and is included mainly for completeness and for comparison purposes. For the inverse Abel transform, other methods are generally more reliable. 
+    3. The :doc:`direct <transform_methods/direct>` numerical integration of the analytical Abel transform equations, which is implemented in Cython for efficiency. In general, while the forward Abel transform is useful, the inverse Abel transform requires very fine sampling of features (lots of pixels in the image) for good convergence to the analytical result, and is included mainly for completeness and for comparison purposes. For the inverse Abel transform, other methods are generally more reliable. 
 
-    4. ``*`` The :doc:`Three Point <transform_methods/three_point>` method of Dasch and co-workers, which provides a fast and robust transform by exploiting the observation that underlying radial distribution is primarily determined from changes in the line-of-sight projection data in the neighborhood of each radial data point. This technique works very well in cases where the real difference between adjacent projections is much greater than the noise in the projections (i.e. where the raw data is not oversampled). 
+    4. ``*`` The :doc:`three_point <transform_methods/three_point>` method of Dasch and co-workers, which provides a fast and robust transform by exploiting the observation that underlying radial distribution is primarily determined from changes in the line-of-sight projection data in the neighborhood of each radial data point. This technique works very well in cases where the real difference between adjacent projections is much greater than the noise in the projections (i.e. where the raw data is not oversampled). 
 
-    5. (Planned implementation) The :doc:`Fourier–Hankel <transform_methods/fh>` method, which is computationally efficient, but contains significant centerline noise and is known to introduce artifacts. 
+    5. The :doc:`two_point <transform_methods/two_point>` method is also well described by Dasch. It is a simpler approximation to the `three point` transform. Computationally, very efficient in Python.
```

Comment:
the two-point and onion-dasch need asterisks to indicate that they need basis sets.